### ISSUE TITLE
Make default test class name pattern more lenient

### ISSUE THREAD

**Issue by CH4Code** [junit-team/junit5#1354]
## Description

junit5 console application only accepts test classes with a _very specific naming convention_. (`*Test`, `*Tests`)

## Issue

`java -jar junit-platform-console-standalone-1.0.0.jar --class-path . -c MenuTest` tests correctly with class:

```java
import static org.junit.jupiter.api.Assertions.assertEquals;
import org.junit.jupiter.api.Test;

class MenuTest {

	@Test
	void testCreateMainWindow() {
		assertEquals(2, 1 + 3);
	}
}
```

but does not find any tests with `java -jar junit-platform-console-standalone-1.0.0.jar --class-path . -c TestMenu` (ofc renaming class and file, re-saving and building the file) 

Copied the jar into the same folder with the .java and .class files.

## Expected

The naming of classes should not matter at all if I EXPLICITLY specify which classes contain tests. At least `Test*` should also be accepted with `--scan-class-path` (which it doesn't).

To resolve this cost me a lot of time btw. Stack overflow is full of helpless questions why the console doesn't find any tests.

## Versions

downloaded `junit-platform-console-standalone-1.0.0.jar` from maven repo. compiled from source `junit-jupiter-api-5.2.0` and `junit-jupiter-engine-5.2.0` and used those to build the tests.

**Comment by sbrannen** [comment]
Tentatively slated for 5.2 M1 for _team discussion_

**Comment by marcphilipp** [comment]
**Team Decision:** Allow `^Test.+` in addition to the existing regex filter.

**Comment by sbrannen** [comment]
This has been addressed in `master` in commit ce303c347e42b606ce51eaf72294dabf19a0ae72.
